contracts/smart-account/modules/SessionValidationModules/ERC20SessionValidationModule.sol

```diff
@@ -3,6 +3,20 @@ pragma solidity 0.8.17;
 import "./ISessionValidationModule.sol";
 import {ECDSA} from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
 
+/**
+ * @title ERC20 Session Validation Modul for Biconomy Smart Accounts.
+ * @dev Validates ERC20 transfers (and approvals) session key signed userOps
+ * Performs basic verifications for every session key signed userOp.
+ * Checks if the session key has been enabled, that it is not due and has not yet expired
```

Comment:
These statements are true for SessionKeyManager, not for validation module. Should be removed from here.

```
Checks if the session key has been enabled, that it is not due and has not yet expired
Then passes the validation flow to appropriate Session Validation module
```

Reply:
fixed thnx